pkg/pipelinerun/description/description.go

```diff
@@ -117,7 +117,11 @@ STARTED	DURATION	STATUS
 {{decorate "results" ""}}{{decorate "underline bold" "Results\n"}}
  NAME	VALUE
 {{- range $result := .PipelineRun.Status.PipelineResults }}
- {{decorate "bullet" $result.Name }}	{{ formatResult $result.Value }}
+{{- if eq $result.Value.Type "string" }}
+ {{decorate "bullet" $result.Name }}	{{ $result.Value.StringVal }}
+{{- else }}
+ {{decorate "bullet" $result.Name }}	{{ $result.Value.ArrayVal }}
+{{- end }}
```

Comment:
0.39 will bring a new type 😅

Reply:
yaa, we may need to handle that

Reply:
should we move ahead with 0.38.3 for now we would want to do a release this week.

Reply:
👍🏼